lib/openc_bot/helpers/register_methods.rb

```diff
@@ -30,6 +31,21 @@ def allowed_hours
         end
       end
 
+      def get_bot_secret(jurisdiction_code=nil, common_secret=nil)
+        Aws.config[:credentials] = Aws::SharedCredentials.new(profile_name: 'bot')
+        client = Aws::SecretsManager::Client.new(region: 'eu-west-2')
+        byebug
```

Comment:
You've left a byebug here. Have you checked this code locally? What tests are there?